roles/edpm_ovs/molecule/default/molecule.yml

```diff
@@ -1,24 +0,0 @@
----
```

Comment:
We should have coverage for the "default" case too, where OVS version hasn't changed. (Basically, for a no-op.) It should be safe (=non-disruptive for DP) to run the task. (I assume that `dnf update` will return 0 and then the role won't do anything. But we should make sure it won't fail.)